bin/check-dns.rb

```diff
@@ -96,15 +96,39 @@ class DNS < Sensu::Plugin::Check::CLI
          long: '--use-tcp',
          boolean: true
 
+  option :count,
```

Comment:
I see thank you for clairifying. I think I prefer something like `--request-count` then, as this is more descriptive. @eheydrick if you agree I will defer to your judgement here. I think count is too generic and I should not HAVE to read the description to understand exactly what something is when a better name could convey it.